lib/fibonacci.rb

```diff
@@ -0,0 +1,22 @@
+# As a developer
+# I want to know the value of a number at an arbitrary position in the
+# Fibonacci sequence
+# so that I can show off my coding skills.
+class Fibonacci
+  attr_accessor :number_in_sequence, :given_number, :sequence
+
+  def initialize(given_number)
+    @given_number = given_number
+    @sequence = [0, 1]
+  end
+
+  def fibonacci
+    if @given_number < @sequence.length
+      return @number_in_sequence = @sequence[@given_number]
+    else
+      l = @sequence.length
+      @sequence << @sequence[l - 1] + @sequence[l - 2]
+      fibonacci
+    end
+  end
+end
```

Comment:
Rubyists usually prefer the full word for variable names: `length = @sequence.length` instead of `l = @sequence.length`.

Since you're using `attr_accessor` on your classes attributes, you can call the attributes without the `@`.

So, line 15, for example, would look like this:
```ruby
return @number_in_sequence = @sequence[@given_number]
```

You can think of `attr_accessor` as a short cut for defining a public method like this:
```ruby
def number_in_sequence
  @number_in_sequence
end
```